src/config.js

```diff
@@ -1,7 +1,8 @@
 const __DEV__ = process.env.NODE_ENV !== 'production'
+const { BASE_URL } = process.env
 
 const __APIPORT__ = 3434
-const __APIURL__ = 'https://media.balthazar.dev/graphql'
+const __APIURL__ = '/graphql'
```

Comment:
i am not too sure about this, but i can't see how this could ever be anything other than localhost

https://www.apollographql.com/docs/link/links/http/#usage uses no base URL here, so i _assume_ it works